lib/src/command/run.dart

```diff
@@ -22,13 +24,20 @@ class RunCommand extends PubCommand {
   RunCommand() {
     argParser.addFlag("checked",
         abbr: "c", help: "Enable runtime type checks and assertions.");
+    argParser.addFlag('list',
```

Comment:
Actually it looks like @nex3 suggested just [--executables](https://github.com/dart-lang/pub/pull/1427#issuecomment-234646423), that sounds good to me as well and is a bit less verbose.